inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/config/CommonConfigHolder.java

```diff
@@ -75,6 +75,17 @@ public class CommonConfigHolder {
     // whether enable whitelist, optional field.
     public static final String KEY_ENABLE_WHITELIST = "proxy.enable.whitelist";
     public static final boolean VAL_DEF_ENABLE_WHITELIST = false;
+    // whether enable file metric, optional field.
+    public static final String KEY_ENABLE_FILEMETRIC = "filemetric.enable";
```

Comment:
It's better to use `file.metric.enable' to make the configuration easy to understand.

Reply:
done